tests/network/localnet/conftest.py

```diff
@@ -256,3 +258,21 @@ def localnet_ovs_bridge_client(
     ) as client:
         assert client.is_running()
         yield client
+
+
+@pytest.fixture()
+def localnet_vms_have_connectivity(localnet_running_vms: tuple[BaseVirtualMachine, BaseVirtualMachine]) -> None:
+    with client_server_active_connection(
+        client_vm=localnet_running_vms[0],
+        server_vm=localnet_running_vms[1],
+        spec_logical_network=LOCALNET_BR_EX_NETWORK,
+    ):
+        pass
+
+
+@pytest.fixture()
+@pytest.mark.usefixtures("localnet_vms_have_connectivity")
+def migrated_localnet_vm(localnet_running_vms: tuple[BaseVirtualMachine, BaseVirtualMachine]) -> BaseVirtualMachine:
```

Comment:
`localnet_vms_have_connectivity` can be called directly in the fixture signature

Reply:
Yes, it can be. However, it is usually recommended to use in the signature fixtures that their values are used in the test body. When the value is not used, it is nicer to use it with `usefixture` to declare its value is not used. I also find it less confusing.